contracts/test/TestLockedAccountMigrationTarget.sol

```diff
@@ -52,9 +52,9 @@ contract TestLockedAccountMigrationTarget is LockedAccount, LockedAccountMigrati
         return true;
     }
 
-    function TestLockedAccountMigrationTarget(IERC667Token _assetToken, Curve _neumarkCurve,
+    function TestLockedAccountMigrationTarget(IERC667Token _assetToken, Curve _neumarkCurve, IAccessPolicy _policy,
```

Comment:
Convention elsewhere is to put forwarded arguments first and in the order of the base classes. `_policy` would come first.